packages/graphql/src/schema-model/library-directives.ts

```diff
@@ -17,13 +17,20 @@
  * limitations under the License.
  */
 import type { Annotations } from "./annotation/Annotation";
+import { annotationsParsers } from "./annotation/Annotation";
 import type { DEPRECATED } from "../constants";
 import { SHAREABLE } from "../constants";
 import type { ValueOf } from "../utils/value-of";
 
 const additionalDirectives = ["alias", "relationship", "relationshipProperties", "node", SHAREABLE] as const;
 
 type LibraryDirectives = keyof Annotations | ValueOf<typeof additionalDirectives> | typeof DEPRECATED;
+export const LIBRARY_DIRECTIVES = [
+    ...(Object.keys(annotationsParsers) as LibraryDirectives[])
```

Comment:
Looks good, my only comment would be to change this type to ` as Array<keyof Annotations>` so it is a bit more accurate